primitives/manta/src/constants.rs

```diff
@@ -59,3 +60,8 @@ pub const MANTA_PAY_PALLET_ID: PalletId = PalletId(*b"mantapay");
 
 /// Default Asset Existential Deposit: Should only be used in TEST
 pub const DEFAULT_ASSET_ED: Balance = 1;
+
+pub mod calamari_staking {
+    // TODO
+    // pub const MinCollatorStake: u32 = 4_000_000;
```

Comment:
please log this todo

Reply:
Postponed to https://github.com/Manta-Network/Manta/issues/798